source/includes/aws/_objects.md

```diff
@@ -403,3 +403,43 @@ Renames a folder in Amazon S3. This operation renames all objects inside the fol
 | `taskId` <br/>*string*     | The task ID related to the folder renaming. |
 | `taskStatus` <br/>*string* | The status of the operation.                  |
 
+<!-------------------- GENERATE URL -------------------->
+
+#### Generate URL
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/rest/services/aws/test-env/objects/:regionName/:bucketName/:pathToObject?&operation=generate_url"
+```
+
+Generates a presigned URL in S3. Must specify a limit in the request which denotes the amount of time (in minutes) until the URL expires. Minimum is 1 minute and maximum is 10008 minutes.
```

Comment:
Exactly 10,008 minutes?  That's hilarious.

Reply:
Oops! It's actually supposed to be 10080. That's the number of minutes in a week xD